src/header/Header.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as React from "react";
+
+// Props for Header component
+type HeaderProps = {
+    className?: string;
```

Comment:
My suggestion is to not use className property to set header color as well. This className is for user define custom class.